app/utils/vr-document-name.js

```diff
@@ -65,22 +65,22 @@ export default class VRDocumentName {
   //   const docType = meta.docType || 'DOC';
```

Comment:
ik weet niet waarom we deze comments bijhouden. Mag ern voor mijn part uit. 
Is niet dat we er iets mee doen of zo he?